Sources/Sentry/Public/SentrySpanProtocol.h

```diff
@@ -87,6 +87,11 @@ NS_SWIFT_NAME(Span)
  */
 - (void)removeTagForKey:(NSString *)key NS_SWIFT_NAME(removeTag(key:));
 
+- (void)setMeasurement:(NSString *)name
+                 value:(NSNumber *)value
+                  unit:(SentryMeasurementUnit *)unit
+    NS_SWIFT_NAME(setMeasurement(name:value:unit:));
```

Comment:
h: This is changing right? Or at least add another function to accept a SentryMeasurement

Reply:
But lets say the user want to keep a list of measurements before passing it to the transaction.

having something like 
```objc
transaction.setMeasurement(someVar)
```
would help a lot, instead of
```objc
transaction.setMeasurement(name: sameVar.name, value: someVar.Value, unit: someVar.unit)
```